### DISCUSSION TITLE
HTML website preview on Web github.dev

### DISCUSSION THREAD

**Discussion by gschurck** [community/community#5879] [Codespaces] [answered]
It would be really usefull if we could just install a live preview extension for HTML. They are all unavailable on github.dev currently.
We can only code without any preview.

**Comment by bamurtaugh** [comment, marked as answer]
Thanks for opening @gschurck. As a point of reference, extensions become available on the github.dev web editor once extension authors follow the steps to create/convert them to a web extension, which is outlined here: https://aka.ms/vscode-web-extensions-guide.

**Reply by gschurck** [reply]
Ok thanks.
